File: codequest/2019Solutions/Lockheed Solutions/Java/Prob04.java
/* CaughtSpeeding.java
 *
 * Copyright (c) 2019 Lockheed Martin (LM) as an unpublished work. 
 * All Rights Reserved.
 *
 * This file does not contain any Lockheed Martin Proprietary Information.
 */

import java.util.Scanner;

public class Prob04 {

	public static void main(String[] args) {
		try (Scanner input = new Scanner(System.in)) {
			int testCases = Integer.parseInt(input.nextLine());

			for (int testcase = 0; testcase < testCases; testcase++) {
				String text = input.nextLine();
				String[] data = text.split(" ");
				
				int speed = Integer.parseInt(data[0]);
				boolean birthday = Boolean.parseBoolean(data[1]);
				
				if(speed <= 60 || (birthday && speed <= 65)) {
					System.out.println("no ticket");
				}
				else if(speed <= 80 || (birthday && speed <= 85)) {
					System.out.println("small ticket");
				}
				else {
					System.out.println("big ticket");
				}
			}
		}
	}

}

File: codequest/2019Solutions/Lockheed Solutions/C/Prob04.c
#include <stdio.h>
#include <math.h>
#include <string.h>



int main() {

    int num_test_cases;
    scanf("%d", &num_test_cases);

    int i;
    for (i = 0; i < num_test_cases; i++) {
    
        int speed;
        char birthday[10];
        scanf("%d %s", &speed, birthday);

        int bd;
        bd = 0;
        if (strcmp(birthday, "true") == 0) {bd = 5;}
  
        if (speed <= 60+bd) {printf("no ticket\n");}
        else if (speed <= 80+bd) {printf("small ticket\n");}
        else {printf("big ticket\n");}
    }
}
